src/main/java/com/CourseScheduleBuilder/controller/RegistrationController.java

```diff
@@ -0,0 +1,45 @@
+package com.CourseScheduleBuilder.controller;
+
+
+import com.CourseScheduleBuilder.Model.User;
+import com.CourseScheduleBuilder.Repositories.UserRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ */
+@RestController
+public class RegistrationController {
+
+    @Autowired //needed to point to the same repo database
+    private UserRepo userrepo;
+
+    @PostMapping("/registration")
+    @CrossOrigin
+    @ResponseBody
+    public boolean login(@RequestBody User user){
```

Comment:
Should this method really be named "login"? It's the registration controller.  This name, since it only appears once, should be called something that describes what this method is doing more precisely.  For example, "validateAndRegisterNewUserRequest"

Reply:
Resolved!